schemas/context/admeasuressearch-adcloud.example.1.json

```diff
@@ -0,0 +1,4 @@
+{
+  "@id": "2419",
```

Comment:
`@id` is special in JSON-LD.  You probably want to be using `xdm:id`

Reply:
Replaced with xdm:id